constants/common.js

```diff
@@ -0,0 +1,12 @@
+
+// Nav
+const EXPLORE_SECTIONS = [
```

Comment:
Don't store routes here. The only file which knows the routes is `routes.js`. You actually don't need this information. Your component `SubMenu` can create link by only passing the name of the route (`explore`) and the param (`{ category: XXX }`).

You have more details in `SubMenu.js`.